cli/main.go

```diff
@@ -57,7 +57,7 @@ func main() {
 	}()
 
 	log.Logger.V(0).Info("starting instrumentors...")
-	if err = inst.Run(ctx); err != nil && err != errors.ErrInterrupted {
+	if err = inst.Run(ctx); err != nil && err != process.ErrInterrupted {
```

Comment:
I would prefer using  https://pkg.go.dev/errors#Is (instead of `!=`) especially that this code is not covered by tests.

Reply:
Sure. I was just preserving existing behavior, but I can add this change to the scope of this PR if there is no objections.